examples/plotting/server/line_animate.py

```diff
@@ -24,6 +24,6 @@ def update(step):
 
 curdoc().add_periodic_callback(update, 50)
 
-session.show() # open the document in a browser
+session.show(p) # open the document in a browser
```

Comment:
To be clear the message would only be necessary if you called it with no args and curdoc().roots is empty. But a message in that case sounds great.